src/reference-designs/tap-networking.md

```diff
@@ -0,0 +1,33 @@
+# Tanzu Application Platform Networking
+
+Tanzu Application Platform Networking diagram describes the Tanzu Application Platform most common cross clusters and external services scenarios networking in multi cluster kubernetes environment. It describes high level egress and ingress networking flows between the Tanzu Application Platform components and with the external services. 
+
+![Tanzu Application Platform networking](img/tap-architecture-planning/tap-networking.jpg)
+
+Table describe the Tanzu Application Platform networking flow, shows in above diagram. 
```

Comment:
The following table describes the networking flow in the Tanzu Application Platform networking diagram.

Reply:
Changes done.